src/eyelinkio/edf/read.py

```diff
@@ -68,6 +68,184 @@ class EDF(dict):
     ----------
     fname : str
         The name of the EDF file.
+
+    Attributes
+    ----------
+    info : dict
+        A :class:`dict` containing information about the EDF file, with keys:
+
+        filename: str
+            The name of the file.
+        meas_date : datetime
+            The date and time that the data was recorded. This datetime is
+            timezone-naive, as the timezone is not stored in the EDF file.
+        version : str
```

Comment:
A little annoying that the EDF file does not store the time zone of the computer it records from.

In our Eyelink ASCII reader in MNE, we forcefully set the date time as UTC, even though we know that the datetime is actually in some local time-zone. I think we did that just to be compliant with MNE's info["meas_date"] standard (must be in UTC).

But here, we just keep the datetime timezone-naive, and I've left a little note about that.

@teonbrooks I'm wondering if you have an opinion on which approach you'd prefer for your data?